prow/overlays/cnv-prod/integration-test-configmap.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: integration-tests
+data:
+  mail-report: "1"
```

Comment:
this would not work in the prod, as we have to send the mail to internal cluster.
turning this off would be better. as we can directly see the results in prow dashboard for period runs.

Reply:
true, a copy&paste error, thx!